frontend/src/assets/css/wrapper.scss

```
@@ -0,0 +1,52 @@
+@import "@assets/css/variables";
+
+.bgImage {
+  position: relative;
+  z-index: 1;
+  display: flex;
+  align-items: center;
+  min-height: 100vh;
+  background-position: center center;
+  background-size: cover;
+}
+
+.bgImage::before {
+  content: "";
+  position: absolute;
+  top: 0;
+  left: 0;
+  z-index: -1;
+  width: 100%;
+  height: 100%;
+  background-color: var(--blue300);
+  opacity: 0.5;
+}
```

Please use SCSS nesting

```suggestion
.bgImage {
  position: relative;
  z-index: 1;
  display: flex;
  align-items: center;
  min-height: 100vh;
  background-position: center center;
  background-size: cover;

  &::before {
    content: "";
    position: absolute;
    top: 0;
    left: 0;
    z-index: -1;
    width: 100%;
    height: 100%;
    background-color: var(--blue300);
    opacity: 0.5;
  }
}
```